apps/bridge-dapp/src/pages/Account/Transactions/TransactionDetail/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { type FC } from 'react';
+import { useNavigate } from 'react-router-dom';
+import {
+  Button,
+  Drawer,
+  DrawerCloseButton,
+  DrawerContent,
+  Typography,
+} from '@webb-tools/webb-ui-components';
+
+import TxDetailContainer from '../../../../containers/TxDetailContainer';
+import { ACCOUNT_TRANSACTIONS_FULL_PATH } from '../../../../constants';
+
+import { randEthereumAddress } from '@ngneat/falso';
+
+const TransactionDetail: FC = () => {
+  const navigate = useNavigate();
+  // const { txHash = '' } = useParams<{ txHash: string }>();
+
+  return (
+    <Drawer
+      defaultOpen
+      onOpenChange={(isOpen) =>
+        !isOpen && navigate(ACCOUNT_TRANSACTIONS_FULL_PATH)
+      }
+    >
+      <DrawerContent className="!w-[500px] flex flex-col justify-between dark:bg-mono-190">
+        {/* Header */}
+        <div className="px-9 pt-9 pb-4 flex items-center justify-between">
+          <Typography variant="h5" fw="semibold">
+            {/* TODO: Update to match with tx activity (Ex: Deposit Details) */}
+            Transaction Details
+          </Typography>
+          <DrawerCloseButton />
+        </div>
+
+        {/* Content */}
+
+        {/* DEPOSIT */}
+        {/* <TxDetailContainer
```

Comment:
Is this meant to be commented out and the withdraw not?

Reply:
It is **intended** here. 

The comment is for me to test in all 3 cases: DEPOSIT, TRANSFER, and WITHDRAW. I commented 2 out of 3 cases here. These will all be removed when integrating with the backend